### DISCUSSION TITLE
Triggering Panda build / PostCSS from within node.js?

### DISCUSSION THREAD

**Discussion by chrislgrigg** [chakra-ui/panda#1334] [Q&A]
I'm attempting to use the new email service [Resend](https://resend.com/), which supports React components and proposes to simplify aspects of sending email from an application. It renders components on the server, so it's best described as SSR. It's not playing well with Panda so far. I'm getting output with classNames but no styles. As far as I can tell it doesn't add any styles because it's rendering from within node, not Next.js's build process, so PostCSS isn't getting involved. Is there a way around this right now?

Looking at how other frameworks/libraries do it, I think the most appropriate thing would be something like `getCssText` from Stitches. Panda's docs [remark](https://panda-css.com/docs/migration/stitches#server-side-rendering) that this isn't necessary because PostCSS takes care of it during build. It looks like it should also be possible for me to invoke PostCSS from node, so maybe there's a way to reuse whatever Panda is already doing on-demand from within the application?

Appreciate your help and advice!

**Comment by anubra266** [comment]
@subvertallchris I've not tried this before, but I'm interested. Just did a quick research to acquaint myself with that space. Are you using `react-email` with resend? or what's your overall stack

**Reply by chrislgrigg** [reply]
Hey, thanks for the quick reply!

I'm using Next.js 13 with the App Router. I just got Resend wired up and it doesn't look like I should need `react-email` directly since it uses it under the hood, though there may be some benefits to using it to compose emails. I did try using the `react-email` `render` function directly to get the stringified output of rendering the component but it has the same problem of missing styles.

I haven't dug too deep and there isn't much in the way of documentation or tips from Resend. It's possible I have overly ambitious expectations and I'll need to fall back to inline styles. 😅
